packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -904,17 +918,16 @@ export function accumulateEventTargetListeners(
   if (eventListeners !== null) {
     const listenersArr = Array.from(eventListeners);
     const targetType = ((event.type: any): DOMTopLevelEventType);
-    const isCapturePhase = (event: any).eventPhase === 1;
```

Comment:
This seems like a significant change. Instead of reading this from the event, we now pass it from outside based on whether it's in a list. Can you please explain why this is a refactoring rather than a semantic change? And what does it accomplish, exactly?

Reply:
I guess there are two things here to note:

- This code is only used with `createEventHandle` logic, so this isn't that much of a significant change given we have no product code anywhere that uses this code path right now.
- It avoids us having to rely on `eventPhase` which can change if you add a listener to a node and the listener triggers on it (which would mean `eventPhase` is `2`, even if it were a capture phase). When we setup the listener we can add this information to the event flags as a detail, completely avoiding this check.